src/qt/test/test_main.cpp

```diff
@@ -41,6 +43,15 @@ Q_IMPORT_PLUGIN(QCocoaIntegrationPlugin);
 #endif
 #endif
 
+static bool UsesXcb() {
+    for (QObject* plugin: QPluginLoader::staticInstances()) {
```

Comment:
How does this react if xcb is available but not default? For ex, if egl or wayland would otherwise be used?